lib/traces/provider.rb

```diff
@@ -21,6 +21,23 @@ def traces_provider
 		end
 	end
 	
+	module Deprecated
+		def trace(...)
+			warn "Traces::Provider.trace is deprecated. Please use Traces.trace instead."
+			Traces.trace(...)
```

Comment:
FYI this makes the minimum Ruby version of this gem be >= 2.7 but that is not reflected in the gemspec